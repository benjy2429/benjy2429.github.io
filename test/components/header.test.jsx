/**
 * @jest-environment jsdom
 */

import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import Header from '../../src/components/header';

describe('Header component', () => {
  it('renders correctly', () => {
    const component = shallow(<Header />);
    expect(toJson(component)).toMatchSnapshot();
  });
});
