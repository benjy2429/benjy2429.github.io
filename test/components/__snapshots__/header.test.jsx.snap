// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header component renders correctly 1`] = `
<section
  className="hero"
>
  <Canvas />
  <div
    className="container"
  >
    <div
      className="hero-wrapper"
    >
      <div
        className="hero-title"
      >
        Hi, I’m Ben Carr.
      </div>
      <div
        className="hero-subtitle"
      >
        I’m a full stack software developer.
      </div>
    </div>
  </div>
</section>
`;
